content/site.json

```diff
@@ -169,7 +169,7 @@
   "footer": {
     "subtitle": "Transforming Data Act Compliance into Scalable Monetization Opportunities",
     "copyright": "Built by [deltaDAO](https://delta-dao.com)",
-    "privacyTitle": "Privacy",
+    "privacyTitle": "LEGAL",
```

Comment:
issue(blocking): lets use normal capitilization here when defining content. The style is then defined in css